types/applicationStage.ts

```diff
@@ -11,6 +11,11 @@ export type ApplicationStageApplicationData = {
 
 export type ApplicationStageData = ApplicationStageApplicationData & { applicationId: number };
 
+export type ApplicationStagePostData = {
```

Comment:
Why is there no emoji and remark here?

They should be here but optional right?

Reply:
We're only creating a new stage here, not adding reactions right? Doesn't make sense to combine the two actions I felt.

1. User creates stage
2. User adds reaction

Reply:
@zognin @ganhongyao wdyt?